build.js

```diff
@@ -45,6 +46,33 @@ const markedOptions = {
   renderer
 }
 
+const htmlMinifierOpts = {
+  collapseBooleanAttributes: true,
+  collapseWhitespace: true,
+  conservativeCollapse: true,
+  decodeEntities: true,
+  minifyCSS: {
```

Comment:
Note to self; see how much inline CSS we have; it shouldn't be a lot, so we could probably set this to `false` too.